src/Argument.h

```diff
@@ -56,13 +56,11 @@ struct Argument {
      * By default, they are left unset, implying "no default, no min, no max". */
     Expr def, min, max;
 
-  int stride_multiples[4];
```

Comment:
Is it possible that this is the first time we have pulled git commits from the Qualcomm Halide git repo via 'git merge' into the hexagon branch?  
Previously we did a 'big blob merge' which did not have all our history, and now we are getting the individual commits whose content matches what is there?